README.md

```diff
@@ -1,75 +1,46 @@
-- Language: Choose a language to search for questions opened recently and without answer
-    - Čeština
-    - Español
-    - English
-    - Magyar
-    - Português (do Brasil)
-- Time in minutes to query the API: set how often the extension should check for new questions posted to the forum
-- Show browser notifications: receive a browser notification when a new question is opened
-- Sidebar you can open the questions in a sidebar
-- Choose one theme to use "Dark" or "Light"
-- Choose one or more products to search for opened questions, the following products are supported:
-    - Firefox for Desktop
-    - Firefox for Enterprise
-    - Firefox for Android
-    - Firefox for iOS
-    - Thunderbird
-- SUMO Live Helper has also been localized in the following languages
-    - Čeština
-    - Español
-    - English
-    - French
-    - Magyar
-    - Português (do Brasil)
-
-If you have any suggestions, would like to see your locale added, or want to see the Add-on localized in your language, feel free to open a PR or an issue. 
-
-## Informations
-1. The search of new opened questions at support forum is in English for default, but you can change in the preferences page and set the time to search for questions as well (the default is each 15 minutes).
-2. The Add-on will show to you the questions that have been opened giving to you the option to choose and open in a new tab.
-
-## Future implementations
-We are looking forward to see what you think is a good idea implement in the future, please to do that open a new [issue](https://github.com/mozillabrasil/sumo_live_helper/issues).
-
-## How to test this Add-on
-1. Clone this repository git clone ```https://github.com/mozillabrasil/sumo_live_helper.git```
-2. In the Firefox address bar type ```about:debugging```
-3. Click ```Load Temporary Add-on```, a dialog window opens
-4. In the cloned repository, select the ```manifest.json``` file. The Add-on icon will appear in your toolbar.
-
-## How Contribute
-Have a suggestion, doubt or encountered an error? Please open a new [issue](https://github.com/mozillabrasil/sumo_live_helper/issues).
-
-Contributions are welcome, feel free to submit a pull :)
-
-## Learn More
-To learn more about webextensions development, see this [Mozilla developer Network documentation](https://developer.mozilla.org/en-US/Add-ons/WebExtensions).
-
-### License
-This repository is available under the [Mozilla Public License 2.0](https://github.com/mozillabrasil/sumo_live_helper/blob/master/LICENSE).
-
-### Credits
-Thank you [philipp-sumo](https://github.com/philipp-sumo/) for your helpfull tips and time to revive this project.
\ No newline at end of file
+
+* Find unanswered questions for Firefox Desktop, Firefox Enterprise, Firefox for
+  Android, Firefox for iOS and Thunderbird
+* Naviguate unanswered questions in a toolbar popup or a sidebar
```

Comment:
I think we can use "Navigate questions using the toolbar popup or sidebar" ... what do you think?

Reply:
Done.